src/libraries/System.Security.Cryptography.Cng/tests/PropertyTests.cs

```diff
@@ -8,6 +8,94 @@ namespace System.Security.Cryptography.Cng.Tests
 {
     public static class PropertyTests
     {
+        private static Lazy<bool> _lazyPlatformCryptoProviderFunctional = new Lazy<bool>(static () =>
+        {
+            CngKey key = null;
+
+            try
+            {
+                key = CngKey.Create(
+                    CngAlgorithm.ECDsaP256,
+                    $"{nameof(PlatformCryptoProviderFunctional)}Key",
+                    new CngKeyCreationParameters
+                    {
+                        Provider = CngProvider.MicrosoftPlatformCryptoProvider,
+                        KeyCreationOptions = CngKeyCreationOptions.OverwriteExistingKey,
+                    });
+
+                return true;
+            }
+            catch (CryptographicException)
+            {
+                return false;
+            }
+            finally
+            {
+                key?.Delete();
+            }
+        }, isThreadSafe: true);
```

Comment:
Thread-safety is the default:
```suggestion
        });
```